lib/releaselib.sh

```diff
@@ -479,25 +476,17 @@ release::gcs::copy_release_artifacts() {
   done
 
   # Copy the main set from staging to destination
-  logecho -n "- Copying release artifacts to $gcs_destination: "
-  logrun -s $GSUTIL -qm cp -r $gcs_stage/* $gcs_destination/ || return 1
+  logecho -n "- Copying public release artifacts to $gcs_destination: "
+  logrun -s $GSUTIL -qm cp -a public_read -r $gcs_stage/* $gcs_destination/ || return 1
 
-  logecho -n "- Marking all uploaded objects public: "
-  logrun -s $LOGRUN_MOCK $GSUTIL -q -m acl ch -R -g all:R \
-                                 "$gcs_destination" || return 1
+  # This small sleep gives the eventually consistent GCS bucket listing a chance
+  # to stabilize before the diagnostic listing. There's no way to directly
+  # query for consistency, but it's OK if something is dropped from the
+  # debugging output.
+  sleep 5
 
```

Comment:
I think we should just revert this PR, then change this not to bother with the `all:R` change. This should be relying on the `defacl` on the bucket and nothing else.